recipes/hippynn/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge hippynn_dev
```

```suggestion
  - conda-forge hippynn_rc
```
See here: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds